bin/beaker-answers

```diff
@@ -4,29 +4,29 @@ require 'rubygems' unless defined?(Gem)
 require 'beaker-answers'
 
 VERSION_STRING =
-"
-                                 _ .--.
-                                ( `    )
-        beaker-answers       .-'      `--,
-                  _..----.. (             )`-.
-                .'_|` _|` _|(  .__,           )
-               /_|  _|  _|  _(        (_,  .-'
-              ;|  _|  _|  _|  '-'__,--'`--'
-              | _|  _|  _|  _| |
-          _   ||  _|  _|  _|  _| %s
-        _( `--.\\_|  _|  _|  _|/
-     .-'       )--,|  _|  _|.`
-    (__, (_      ) )_|  _| /
-      `-.__.\\ _,--'\\|__|__/
-                    ;____;
-                     \\YT/
-                      ||
-                     |\"\"|
-                     '=='
-Supports: %s
-"
+  "
+                                   _ .--.
+                                  ( `    )
+          beaker-answers       .-'      `--,
+                    _..----.. (             )`-.
+                  .'_|` _|` _|(  .__,           )
+                 /_|  _|  _|  _(        (_,  .-'
+                ;|  _|  _|  _|  '-'__,--'`--'
+                | _|  _|  _|  _| |
+            _   ||  _|  _|  _|  _| %s
+          _( `--.\\_|  _|  _|  _|/
+       .-'       )--,|  _|  _|.`
+      (__, (_      ) )_|  _| /
+        `-.__.\\ _,--'\\|__|__/
+                      ;____;
+                       \\YT/
+                        ||
+                       |\"\"|
+                       '=='
+  Supports: %s
+  ".freeze
 
-versions = BeakerAnswers::Answers.supported_versions.map{ |v| v.to_s }.join(', ')
-puts VERSION_STRING % [BeakerAnswers::Version::STRING, versions]
```

Comment:
Drop this one.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleformatstring disabled